butte-build/src/codegen.rs

```diff
@@ -0,0 +1,559 @@
+use crate::types::*;
```

Comment:
Maybe we should consider a prelude (see `std::io::prelude`)

Reply:
I'm not sure, since IIUC a prelude is useful for external users, and the only thing that this library exposes is the `compile_fbs` function. Everything else has `pub(crate)` visibility.